### PR TITLE
Ft Stripe - Payment Intent PR

### PR DESCRIPTION
**Pull request by ChiragThesia**
## Description
Created payment intent for stripe. Look at Paymentrouter.js

## Checklist
Remove any items which are not applicable.
- [X] I have performed a self-review of my own code
- [X] My code follows the style guidelines of this project
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works .
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: controllers/seller/getSellerFromStore.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+const Seller = require('../../models/seller');
+const Store = require('../../models/store');
+
+const getSellerFromStore = async (req, res) => {
+	try {
+		const { storeId } = req.params;
+		console.log('StoreID?', storeId);
```
Comment (Cireimu) [inline controllers/seller/getSellerFromStore.js:7]: Remove this console log if it's not needed

File: controllers/seller/getSellerFromStore.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+		const store = await Store.findOne({ _id: storeId });
+		console.log('Is there a store', store);
```
Comment (Cireimu) [inline controllers/seller/getSellerFromStore.js:10]: Remove this console log if it's not needed

File: controllers/seller/getSellerFromStore.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+		const sellerId = store.seller;
+		const seller = await Seller.findOne({ _id: sellerId });
+		console.log('Seller?', seller);
```
Comment (Cireimu) [inline controllers/seller/getSellerFromStore.js:14]: Remove this console log if it's not needed

File: routes/paymentRouter.js [hunk 4]
```diff
@@ -1,80 +1,39 @@
...
-    , {
-      stripeAccount: stripeId
-    }
-    )
-    res.status(200).json({ paymentIntent: paymentIntent, stripeId })
-  } catch (error) {
-    res.status(400).json(error)
-  }
-})
+		console.log('paymentIntent', paymentIntentActual);
```
Comment (Cireimu) [inline routes/paymentRouter.js:24]: Remove this console log if it's not needed

Comment (ChiragThesia) [reply routes/paymentRouter.js]: this should be removed now.

File: controllers/seller/getSellerFromStore.js (hunk 3 above)
Comment (ChiragThesia) [reply controllers/seller/getSellerFromStore.js]: Should be up to date now.

**Pull request merged**